dashboard/app/views/pd/regional_partner_contact/new.html.haml

```diff
@@ -8,6 +8,13 @@
   %h2
     Contact a regional partner
 
+.form-group
+  .question School Country
+  %select#us-or-international.form-control{ name: "school_info[country]", type: "select", required: true, onchange: "if (this.value === 'International') {window.location = 'https://support.code.org/hc/en-us/articles/202518373-How-do-I-bring-computer-science-to-my-country-'}"}
```

Comment:
can we do `window.open` instead? Force-redirecting when they might be in the middle of filling out a form is pretty bad manners.